lib/src/view/game/game_common_widgets.dart

```diff
@@ -61,20 +124,10 @@ class GameAppBar extends ConsumerWidget {
               : const SizedBox.shrink(),
       actions: [
         const ToggleSoundButton(),
-        if (id != null)
-          AppBarIconButton(
-            onPressed:
-                () => showAdaptiveBottomSheet<void>(
-                  context: context,
-                  isDismissible: true,
-                  isScrollControlled: true,
-                  showDragHandle: true,
-                  constraints: BoxConstraints(minHeight: MediaQuery.sizeOf(context).height * 0.5),
-                  builder: (_) => GameSettings(id: id!),
-                ),
-            semanticsLabel: context.l10n.settingsSettings,
-            icon: const Icon(Icons.settings),
-          ),
+        if (id != null) ...[
```

Comment:
I'd rather not have the bookmark button as a separate button in the app bar. It shouldn't be that prominent.

2 possibilities:
- make a drop down menu (like in study screen) that has settings and bookmark (and sound probably, that would be removed from settings)
- put the bookmark in the game menu (from the bottom bar)